### PR TITLE
[luci] Add version control codes in FuseBCQPass

### PR DESCRIPTION
**Pull request by llFreetimell**
Parent Issue : #3481
Draft : #3329

This commit will add version control codes in `FuseBCQPass`.

ONE-DCO-1.0-Signed-off-by: Seok NamKoong <seok9311@naver.com>

### CODE REVIEW

File: compiler/luci/pass/src/FuseBCQPass.cpp [hunk 1]
```diff
@@ -431,7 +431,39 @@ bool FuseBCQPass::run(loco::Graph *g)
...
+
+  // If version node is not found, regard it as version 1.
+  int32_t bcq_version = (version_node == nullptr) ? version_node->at<loco::DataType::S32>(0) : 1;
+
+  if (bcq_version == 1)
+    changed = BCQFuser<1>().fuseBCQ(g);
+  else
+    assert(false && "Not supported BCQ version");
+
+  if (changed && !version_node)
```
Comment (seanshpark) [inline compiler/luci/pass/src/FuseBCQPass.cpp:462]: `!version_node` ? maybe `if (changed && version_node != nullptr)`

Comment (llFreetimell) [reply compiler/luci/pass/src/FuseBCQPass.cpp:462]: oops.... my terrible mistake :(

Comment (llFreetimell) [inline compiler/luci/pass/src/FuseBCQPass.cpp:462]:
```suggestion
  if (changed && version_node != nullptr)
```

**Comment by llFreetimell** [comment]
@chunseoklee Can you merge this? :)

**Pull request merged**